monero-outreach-docs/translations/es/archive/getting_started_es.md

```diff
@@ -0,0 +1,197 @@
+_¿Dónde empezar?_ [taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation]](https://taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation/)
+
+#### Soporte
+Ayudando a los usuarios por una variedad de razones, incuye la configuración de monederos, la conexión a nodos, restauración de cuentas, transferencia de fondos y más.
+_¿Dónde empezar?_ [r/MoneroSupport](https://www.reddit.com/r/MoneroMining/), #monero-support y [monero.stackexchange.com](https://monero.stackexchange.com/)
+
+#### Minería
+Asegurando la cadena de bloques de Monero. Incluye discusiones sobre el lanzamiento de nuevos desarrollos de software y guías útiles para la minería en variedad de configuraciones de software y hardware.  
+
+#### Trading
+Discusiones sobre XMR y temas en general sobre especulación del precio.
+_¿Dónde empezar?_ [r/xmrtrader](https://www.reddit.com/r/xmrtrader/) y #monero-markets
+
+#### GUI
+Mejorando la interfaz gráfica del usuario para el software de Monero.
+_¿Dónde empezar?_ [github.com/monero-project/monero-gui](https://github.com/monero-project/monero-gui), #monero-gui y #monero-dev
+
+#### Proceso de respuesta a vulnerabilidades
+Manejo del proceso de respuesta a vulnerabilidades. Se debe aceptar invitación de parte del equipo de trabajo.
+_¿Dónde comenzar?_ [github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md)
+
+#### RIAT - Instituto para la Economía futura
+Promoviendo proyectos de cryptomonedas, incluyendo a Monero. Instituto basado en Vienna.
+_¿Dónde comenzar?_ [https://riat.at](https://riat.at/)
+
+#### Defcon – Monero Village
+Planeación y ejecución del evento anual. (Estacional)
+_¿Dónde empezar?_ [taiga.getmonero.org/project/michael-defcon-vegas](https://taiga.getmonero.org/project/michael-defcon-vegas/) y #monero-defcon
+
+#### CCC (Chaos Computer Club)
+Planeación y ejecución del evento anual. (Estacional)
+_¿Dónde empezar?_ [taiga.getmonero.org/project/rehrar-35c3-2](https://taiga.getmonero.org/project/rehrar-35c3-2/)
+
+#### Mastering Monero (Libro - Dominando a Monero)
+Mejorando y actualizando futuras ediciones del libro financiado por la comunidad.
+_¿Dónde empezar?_ [github.com/monerobook/monerobook](https://github.com/monerobook/monerobook)
+
+#### Kit de encuentro de Monero
+Elaboración de herramientas para kits de encuentros de Monero.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/sgp-monero-meetup-kit](https://taiga.getmonero.org/project/sgp-monero-meetup-kit/)
+
+#### Integraciones
+Colaborar para poder integrar a Monero en una variedad de servicios, esto incluye procesadores de pagos, casas de cambio descentralizadas y más.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/serhack-monero-integrations](https://taiga.getmonero.org/project/serhack-monero-integrations/) & #monero-integrations
+
+#### Labratorio de transmisiones de Monero
```

Comment:
It is **Laboratorio**, not Labratorio, it's a typo error.

Reply:
Oups, thanks xD